devcontainers/lib/dependabot/devcontainers/metadata_finder.rb

```diff
@@ -0,0 +1,22 @@
+# typed: true
+# frozen_string_literal: true
+
+require "dependabot/metadata_finders"
+require "dependabot/metadata_finders/base"
+
+module Dependabot
+  module Devcontainers
+    class MetadataFinder < Dependabot::MetadataFinders::Base
+      private
+
+      def look_up_source
+        # TODO: Make upstream changes to dev container CLI to point to docs.
+        #       Specifically, 'devcontainers features info' can be augmented to expose documentationUrl
+        nil
```

Comment:
PRs would have no release or changelog info for now. Seems fine to get started but just noticing it.

Reply:
👍  I have an idea of what to put here, but wasn't sure how to easily source it - I think making an upstream change to the devcontainers CLI to fetch this info would be the best approach (don't want to add too much devcontainer-specific logic here in core)